crates/sui-rpc-api/proto/sui/rpc/v2alpha/move_descriptor.proto

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package sui.rpc.v2alpha;
+
+// Describes a data type.
+message Descriptor {
+  // Name of this DataType
+  optional string name = 1;
+
+  // XXX Do we want both storage/runtime info here??
```

Comment:
Maybe this is what you're alluding to, but the package ID returned here should always be the "defining ID" for the type in question. So, even if you are describing a module at version V of a package, if it contains a type introduced at version `U < V`, then this should mention `U`, not `V`.